.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -40,12 +40,12 @@ body:
       label: Expected Behavior
       description: |
         What did you expect to happen? For example:
-        - I expected that the PowerShell script wouldn't give me this error message.
+        - I expected the PowerShell script wouldn't give me this error message.
         - I expected that the _Windows Sandbox_ wouldn't crash.
-        - I expected that the _Lockdown Browser_ wouldn't detect that it's running inside of a virtual machine.
-        - I expected that my webcam would work properly inside the _Windows Sandbox_.
+        - I expected that the _Lockdown Browser_ wouldn't detect that it's running inside a virtual machine.
+        - I expected my webcam to work properly inside the _Windows Sandbox_.
 
-        If you can't describe what _should_ happen, then this is probably a question and is better suited for a [discussion](https://github.com/gucci-on-fleek/lockdown-browser/discussions/new).
+        If you can't describe what _should_ happen, this is probably a question and better suited for a [discussion](https://github.com/gucci-on-fleek/lockdown-browser/discussions/new).
```

1) I've taken too much logic to tolerate an “if” without a “then” :)

2) "suited" is either a past participle (in which case it needs an auxiliary verb) or an adverb (in which case it needs a verb to modify), so I think that we need to keep the “is” here. (Sorry, I've taken _way_ too many grammar classes)
```suggestion
        If you can't describe what _should_ happen, then this is probably a question and is better suited for a [discussion](https://github.com/gucci-on-fleek/lockdown-browser/discussions/new).
```